frodo-runtime/src/main/java/com/fernandocejas/frodo/aspect/LogSubscriber.java

```diff
@@ -78,9 +80,14 @@ public void beforeOnStartExecution(JoinPoint joinPoint) {
   public void beforeOnNextExecution(JoinPoint joinPoint) {
     countAndMeasureTime();
     final FrodoJoinPoint frodoJoinPoint = new FrodoJoinPoint(joinPoint);
-    final Object value = frodoJoinPoint.getMethodParamValuesList().get(0);
-    messageManager.printSubscriberOnNext(joinPoint.getTarget().getClass().getSimpleName(),
-        value.toString(),
+    final String message;
```

Comment:
@android10 So you suggest to pass the whole object list `frodoJoinPoint.getMethodParamValuesList()` in the method. I am saying that because if we don't pass it and keep the logic `Object value = frodoJoinPoint.getMethodParamValuesList().get(0)` then we are going to have another exception `ArrayIndexOutOfBoundsException`